site/_includes/partials/chrome-100-card-section.njk

```diff
@@ -0,0 +1,2584 @@
+        <li id="menu__section2012" class="menuitem">
+          <a class="sectionLink" data-index="4" href="#section2012" data-label="2012"
+              data-category="year nav" data-action="nav click">2012</a>
+        </li>
+        <li id="menu__section2013" class="menuitem">
+          <a class="sectionLink" data-index="5" href="#section2013" data-label="2013"
+              data-category="year nav" data-action="nav click">2013</a>
+        </li>
+        <li id="menu__section2014" class="menuitem">
+          <a class="sectionLink" data-index="6" href="#section2014" data-label="2014"
+              data-category="year nav" data-action="nav click">2014</a>
+        </li>
+        <li id="menu__section2015" class="menuitem">
+          <a class="sectionLink" data-index="7" href="#section2015" data-label="2015"
+              data-category="year nav" data-action="nav click">2015</a>
+        </li>
+        <li id="menu__section2016" class="menuitem">
+          <a class="sectionLink" data-index="8" href="#section2016" data-label="2016"
+              data-category="year nav" data-action="nav click">2016</a>
+        </li>
+        <li id="menu__section2017" class="menuitem">
+          <a class="sectionLink" data-index="9" href="#section2017" data-label="2017"
+              data-category="year nav" data-action="nav click">2017</a>
+        </li>
+        <li id="menu__section2018" class="menuitem">
+          <a class="sectionLink" data-index="10" href="#section2018" data-label="2018"
+              data-category="year nav" data-action="nav click">2018</a>
+        </li>
+        <li id="menu__section2019" class="menuitem">
+          <a class="sectionLink" data-index="11" href="#section2019" data-label="2019"
+              data-category="year nav" data-action="nav click">2019</a>
+        </li>
+        <li id="menu__section2020" class="menuitem">
+          <a class="sectionLink" data-index="12" href="#section2020" data-label="2020"
+              data-category="year nav" data-action="nav click">2020</a>
+        </li>
+        <li id="menu__section2021" class="menuitem">
+          <a class="sectionLink" data-index="13" href="#section2021" data-label="2021"
+              data-category="year nav" data-action="nav click">2021</a>
+        </li>
+        <li id="menu__section2022" class="menuitem">
+          <a class="sectionLink" data-index="14" href="#section2022" data-label="2022"
+              data-category="year nav" data-action="nav click">2022</a>
+        </li>
+      </ul>
+    </nav>
+  </div>
+
+  <div class="card-container">
+    {# Year 2008 #}
+    <section class="inner-card display-flex center-margin gap-right-300 year-2008" id="section2008" data-index="0">
+      <div class="first-column gap-right-300">
+        <div class="card card-with-img gap-bottom-300">
+          {% Img src="image/x1Los57vDga6OEMNi1dIJwZ0qvp2/KqP5cQtSMZhtKXw93XcW.png", alt="card image 2008", class="card-img", width="440", height="170" %}
```

Comment:
Please use meaningful alt text instead of placeholder "card image xxx"

Reply:
Done